app/script/conversation/ConversationRepository.js

```diff
@@ -825,36 +825,46 @@ z.conversation.ConversationRepository = class ConversationRepository {
 
   /**
    * Get conversation with a user.
-   * @param {User} user_et - User entity for whom to get the conversation
-   * @param {string} [team_id] - Team ID in which the conversation should be searched
+   * @param {User} userEntity - User entity for whom to get the conversation
    * @returns {Promise} Resolves with the conversation with requested user
    */
-  get_1to1_conversation(user_et, team_id = this.team().id) {
-    for (const conversation_et of this.conversations()) {
-      const with_expected_user = user_et.id === conversation_et.participating_user_ids()[0];
+  get1to1Conversation(userEntity) {
```

Comment:
We might want to name it `get1To1Conversation` in order to be consistent (have an uppercase after a number)